manifests/4.1/image-references

```diff
@@ -0,0 +1,24 @@
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+spec:
+  tags:
+  - name: cluster-logging-operator
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/origin-cluster-logging-operator:latest
+  - name: logging-kibana5
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/origin-logging-kibana5:latest
+  - name: logging-curator5
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/origin-logging-curator5:latest
+  - name: logging-fluentd
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/origin-logging-fluentd:latest
+  - name: rsyslog
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/origin-logging-rsyslog:latest
```

Comment:
No need to be `docker.io/viaq/rsyslog:latest` any more?

If we search, these 2 hit in the CLO code.  I'd think the first one is (or both of them could be) obsolete???
```
Makefile:	RSYSLOG_IMAGE=quay.io/viaq/rsyslog:latest \
controller-manifests/cluster-logging.v4.1.0.clusterserviceversion.yaml:                    value: "docker.io/viaq/rsyslog:latest"
```

Reply:
@richm I'm removing this from the imageref file since rsyslog will not be in 4.1  We will have to add it in for 4.2